### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by l-koliesnik**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        <ul class="horizontal-list">
+            <li><a href="#howitworks"><span id="word-spacing">How it works</span></a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contacts">Contacts</a></li>
+        </ul>
+    </div>
+    <div class="one">
+        <div class="head">
+            <div id="headline">
+                <h1>SAFOR<br>
```
Comment (ghost) [inline index.html]: Please don't use `<br>`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+            <p id="threelongp">Set up your home and office locations to simplify your search in the city.
+                Add your family contact to notify them in dangerius situation</p>
+        </div>
+        <div>
+            <img src="Bitmap.png" id="threeimg1">
+            <img src="Bitmap2.png" id="threeimg2">
+        </div>
+        <div class="four">
+            <h3>WHY SAFOR?</h3>
+            <h2>Reliability</h2>
```
Comment (ghost) [inline index.html]: There better use list for containing items

File: index.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+            <h2>Anonymity</h2>
+            <p>Your relatives can be notified that you need help only with your consent</p>
+        </div>
+    </div>
+    <div class="five">
+        <div>
+            <div class="forbusiness">FOR BUSINESS</div>
+        </div>
+        <div id="fivediv">FOR PEOPLE</div>
+        <div id="fivetext">
```
Comment (ghost) [inline index.html]: Use list for this too

**Review by ghost — CHANGES_REQUESTED** [review]
Adaptive is not implemented for 1024px and less

File: index.html [hunk 4]
```diff
@@ -114,6 +153,14 @@ <h1>HAVE QUESTIONS?<br>
             </form>
         </div>
     </div>
+    <div class="footer-tablet">
```
Comment (ghost) [inline index.html]: Please don't create tablet version of divs, do not repeat code, just style previous one with css.

File: index.html [hunk 5]
```diff
@@ -102,6 +103,44 @@ <h2>Anonymity</h2>
                 We use all available resources to achieve maximum results in our work and communication.</p>
         </div>
     </div>
+    <div class="fivefortablet">
```
Comment (ghost) [inline index.html]: Why do we need version for tablet ? It can be changed using css.

**Review by ghost — CHANGES_REQUESTED** [review]

File: index.html [hunk 6]
```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html>
+
```
Comment (volo-maslow) [inline index.html]: The spacing here is useless, please remove all newlines between elements.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./assets/style.css" type="text/css" rel="stylesheet">
```
Comment (volo-maslow) [inline index.html]: Do not use ./ to reference a file (styles or js), just use the folder name or file itself. (assets/style.css).
Change for all href's.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+            <li><a href="#howitworks"><span id="word-spacing">How it works</span></a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contacts">Contacts</a></li>
+        </ul>
+    </div>
+    <div class="one">
+        <div class="head">
+            <div class="headline">
+                <h1 id="headlineSAFOR">SAFOR</h1>
+                <h1>switching your<br>
```
Comment (volo-maslow) [inline index.html]: Please remove all BR's, this should be working with single text inside without any inner tags.

File: index.html (hunk 1 above)
Comment (ghost) [reply index.html]: Still reproduces in some lines of code

File: index.html [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+                <hr id="hr2">
+                <hr id="hr3">
+                <hr id="hr4">
+                <hr id="hr5">
+                <hr id="hr6">
+                <hr id="hr7">
+            </div>
+        </div>
+    </div>
+    <div class="two">
```
Comment (volo-maslow) [inline index.html]: Change class names to be more descriptive, for example - main-content, or footer-feedback, or anything else representing the content inside the div or section.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+<body>
+    <div class="header">
+        <img src="assets/logo.png" class="logo" id="logo">
+        <ul class="horizontal-list">
+            <li><a href="#howitworks"><span id="word-spacing">How it works</span></a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contacts">Contacts</a></li>
+        </ul>
+    </div>
+    <div class="one">
```
Comment (volo-maslow) [inline index.html]: Please use section instead of div's here, top-level divs should be a section.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,146 @@
...
+            </div>
+        </div>
+    </div>
+    <div class="two">
+        <div id="about">
+            <div>
+                <h1>WHAT IS SAFOR?</h1>
+                <div>
+                    <img src="assets/Rectangle.svg">
+                    <h3 id="abouth3"><a href=" ">WATCH NOW</a></h3>
```
Comment (volo-maslow) [inline index.html]: Please use underscore or dash to separate two different tags. Same for all Id's and Classes in document.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,146 @@
...
+                </div>
+            </div>
+        </div>
+        <div class="aboutsafor">
+            <p> The existing types of protection in today's society can not always
+                be adapted to the needs of the modern customers.
+                <br><br>
```
Comment (volo-maslow) [inline index.html]: Do not use `<br>` inside the text, it is not possible to update such field from admin panel, users will not enter a `<br>` tag inside their text, they will have just a raw text instead.

File: index.html [hunk 13]
```diff
@@ -0,0 +1,146 @@
...
+                urbanjungle
+            </p>
+            <h2>Speed</h2>
+            <p>Do not waste time explaining the situation and describing what is happening</p>
+            <h2>Anonymity</h2>
+            <p>Your relatives can be notified that you need help only with your consent</p>
+        </div>
+    </div>
+    <div class="five">
+        <div class="forbusiness" id="forbusiness">
```
Comment (volo-maslow) [inline index.html]: ID here is useless, please use only a class. Also please review the document, a lot of parts doesn't seem to be needed (especially a combo with id & class with the same name).

File: index.html [hunk 14]
```diff
@@ -0,0 +1,146 @@
...
+            <p id="threelongp">Set up your home and office locations to simplify your search in the city.
+                Add your family contact to notify them in dangerius situation</p>
+        </div>
+        <div>
+            <img src="assets/Bitmap.png" id="threeimg1">
+            <img src="assets/Bitmap2.png" id="threeimg2">
+        </div>
+        <div class="four">
+            <h3>WHY SAFOR?</h3>
+            <h2>Reliability</h2>
```
Comment (volo-maslow) [inline index.html]: This should be done with UL & LI elements.

**Review by volo-maslow — CHANGES_REQUESTED** [review]

**Pull request closed without merging**